src/cui.cpp

```diff
@@ -199,6 +200,11 @@ void Line::log() {
             << "\t" << recv_value << std::endl;
 }
 
+void Line::log_json() {
+  printf("{\"name\":\"%s\",\"PID\":%d,\"UID\":%d,\"RX\":%f,\"TX\":%f }",
+        m_name, m_pid, m_uid, recv_value, sent_value);
```

Comment:
with `-v` the user can select a 'view mode' (showing kb/s, kb, b or mb) - but it seems reasonable to ignore this for the json output